src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -700,7 +700,7 @@ private static JsonDocument Parse(
             PooledByteBufferWriter? extraPooledByteBufferWriter = null)
         {
             ReadOnlySpan<byte> utf8JsonSpan = utf8Json.Span;
-            var database = MetadataDb.CreateRented(utf8Json.Length, convertToAlloc: false);
+            var database = MetadataDb.CreateRented(utf8JsonSpan.Length, convertToAlloc: false);
```

Comment:
Does this make a difference?

Reply:
Not sure why I changed this one, the other places are actually skipping the indirection through `Memory<T>`.